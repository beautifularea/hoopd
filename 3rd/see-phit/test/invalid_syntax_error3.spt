R"*(
<HTML>
  <P> Space before void tag close angle bracket</P>
  <IMG/ >
</HTML>
)*"_html; 