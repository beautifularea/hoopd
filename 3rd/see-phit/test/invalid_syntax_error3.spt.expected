Error() [with int ROW = 4; int COL = 8; WHAT = spt::Missing_close_bracket_on_void_tag]':
